docusaurus/docs/Android/04-compose/04-message-components/04-message-composer.mdx

```diff
@@ -13,13 +13,6 @@ The **bound** version of the composer uses the **stateless** composer internally
 Additionally, we cannot provide a default `ViewModel`, as it requires the `channelId` to send data, so you'll have to build an instance yourself.
```

Comment:
Not related, but the above line "send messages and attachments to the chat, participating in conversation." sounds weird